viite-UI/src/view/navigationpanel/ProjectListModel.js

```diff
@@ -63,6 +63,7 @@
           html += '</table>';
           $('#project-list').html($(html));
           $('[id*="open-project"]').click(function(event) {
+            eventbus.trigger("Opening_road_address_project_" + event.currentTarget.value);
```

Comment:
Trigger name=???

Reply:
That was something I added in order to get some feedback from the application itself when clicking the button (I had to hope it was working to know something did not break) removing in next commit.